index.js

```diff
@@ -38,7 +38,10 @@ Settings.loadRepositoriesSettings('./config.json')
                 'User-Agent': 'cesium-concierge',
                 Authorization: 'token ' + repositorySettings.gitHubToken
             };
-            var checkChangesMd = defined(repositorySettings.checkChangesMd) ? repositorySettings.checkChangesMd : true;
+            var checkChangesMd = false;
+            if (defined(repositorySettings.checkChangesMd)) {
+                checkChangesMd = (repositorySettings.checkChangesMd === 'true');
```

Comment:
I sent you an email with a link to a more complete parsing function you can use here.